docs/make.jl

```diff
@@ -0,0 +1,23 @@
+using HealthSampleData
+using Documenter
+
+DocMeta.setdocmeta!(HealthSampleData, :DocTestSetup, :(using HealthSampleData); recursive=true)
+
+makedocs(;
+    modules=[HealthSampleData],
+    authors="TheCedarPrince <jacobszelko@gmail.com> and contributors",
+    sitename="HealthSampleData.jl",
+    format=Documenter.HTML(;
+        canonical="https://TheCedarPrince.github.io/HealthSampleData.jl",
+        edit_link="master",
+        assets=String[],
+    ),
+    pages=[
+        "Home" => "index.md",
+    ],
+)
+
+deploydocs(;
+    repo="github.com/TheCedarPrince/HealthSampleData.jl",
+    devbranch="master",
+)
```

Comment:
```suggestion
makedocs(;
    modules=[HealthSampleData],
    authors="TheCedarPrince <jacobszelko@gmail.com> and contributors",
    sitename="HealthSampleData.jl",
    format=Documenter.HTML(;
        canonical="https://TheCedarPrince.github.io/HealthSampleData.jl",
        edit_link="master",
        assets=String[],
    ),
    pages=[
        "Home" => "index.md",
    ],
)

deploydocs(;
    repo="github.com/TheCedarPrince/HealthSampleData.jl",
    devbranch="master",
)
```

This is almost right! However, please update the format and deployment based on this: https://github.com/JuliaHealth/OMOPCDMFeasibility.jl/blob/master/docs/make.jl You can ignore the pages as so far there is only one page.